src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return totalCost - 20;
```

Comment:
Consider extracting the magic number '20' to a constant variable named 'smallDiscount', for better code readability and maintainability. 😊

Reply:
Fixed.